params/bootnodes.go

```diff
@@ -38,9 +38,9 @@ var GardenBootnodes = []string{
 // OrchardBootnodes are the enode URLs of the P2P bootstrap nodes running on the
 // Orchard test network
 var OrchardBootnodes = []string{
-	"enode://f99403abcfbee37e3232e6bb4d7fda4d70496585a53857ccb4aff5ec32d0f27186b5097430d5806f20f2003f35cfec5d778598a3945d530f212b7072caab9b8a@35.188.17.207", // us-central1-b
+	"enode://f99403abcfbee37e3232e6bb4d7fda4d70496585a53857ccb4aff5ec32d0f27186b5097430d5806f20f2003f35cfec5d778598a3945d530f212b7072caab9b8a@35.188.17.207",  // us-central1-b
```

Comment:
Did you mean to change the bootnodes? I'd be surprised this is related to blake3 vs progpow

Reply:
didnt change the bootnodes, just the formatter added a space I think